spec/models/manager_refresh/persister/serializing_spec.rb

```diff
@@ -0,0 +1,124 @@
+require_relative '../helpers/spec_parsed_data'
+require_relative 'test_persister'
+require_relative 'targeted_refresh_spec_helper'
+
+describe ManagerRefresh::Inventory::Persister do
+  include SpecParsedData
+  include TargetedRefreshSpecHelper
+
+  ######################################################################################################################
+  # Spec scenarios testing Perister can serialize/deserialize, with having complex nested lazy_find links
+  ######################################################################################################################
+  #
+  before :each do
+    @zone = FactoryGirl.create(:zone)
+    @ems  = FactoryGirl.create(:ems_cloud,
+                               :zone            => @zone,
+                               :network_manager => FactoryGirl.create(:ems_network, :zone => @zone))
+
+    allow(@ems.class).to receive(:ems_type).and_return(:mock)
+    allow(Settings.ems_refresh).to receive(:mock).and_return({})
+  end
+
+  let(:persister) { create_persister }
```

Comment:
Do you need this?  You're setting it to a local var [here](https://github.com/ManageIQ/manageiq/pull/17361/files#diff-fca70a52058ca7e482e973ed35dad096R26)

Reply:
right, I don't use that one, let me delete it